hack/make/tgz

```diff
@@ -25,6 +25,9 @@ for d in "$CROSS/"*/*; do
 		# if windows use a zip, not tgz
 		BUNDLE_EXTENSION=".zip"
 		IS_TAR="false"
+	elif [ "$GOOS" == "solaris" ]; then
```

Comment:
@LK4D4 : The libnetwork changes that I vendored in with my original PR were being called only from Solaris files. This caused an issue with the vendor CI which being run on a linux box did not detect the dependencies in use by Solaris files. 
To fix this I added the solaris/amd64 tag in vendor. As a result of this make/cross tried to build Solaris binaries and failed due to the CGO calls which call into Solaris specific header files.
So to make the vendor CI and make/cross pass we added bypassed the cross build for Solaris with the above elif continue.